gating/periodic/upload_test_results_to_qtest.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+## Shell Opts ----------------------------------------------------------------
+
+set -x
+
+## Vars ----------------------------------------------------------------------
+
+export QTEST_API_TOKEN=${RPC_ASC_QTEST_API_TOKEN}
+VENV_NAME="venv-qtest"
+VENV_PATH="${WORKSPACE}/${VENV_NAME}"
+ZIGZAG_PYPI_PACKAGE="rpc-zigzag~=1.0"
+ZIGZAG_CONFIG_FILE="asc_sys-tests_zigzag_config.json"
+OSP_VERSION="osp${REDHAT_OSP_VERSION}"
+PROJECT_ID="85664"
+
+## ZigZag Config -------------------------------------------------------------
+cat <<-EOF >$ZIGZAG_CONFIG_FILE
+{
+  "zigzag": {
+    "test_cycle": "$OSP_VERSION",
+    "project_id": "$PROJECT_ID",
+    "module_hierarchy": [
+      "{{ JOB_NAME }}",
+      "{{ MOLECULE_TEST_REPO }}.{{ MOLECULE_SCENARIO_NAME }}",
+      "{{zz_testcase_class}}"
+    ],
+    "path_to_test_exec_dir": "/molecule/{{ MOLECULE_SCENARIO_NAME }}",
+    "build_url": "{{ BUILD_URL }}",
+    "build_number": "{{ BUILD_NUMBER }}",
+    "test_repo_name": "{{ MOLECULE_TEST_REPO }}",
+    "test_branch": "{{ RE_JOB_BRANCH }}",
+    "test_fork": "rcbops",
+    "test_commit": "{{ MOLECULE_GIT_COMMIT }}"
+  }
+}
+EOF
+
+## Functions -----------------------------------------------------------------
+
+source $(dirname ${0})/../../scripts/functions.sh
```

Comment:
There is no such directory for file in this repo.

Reply:
@johnduarte Updated

Reply:
@zreichert does the lack of that file prevent a needed environment variable from being set?  I was unable to understand the value of its inclusion.

Reply:
It was a bit of boilerplate that was copied from rpc-openstack.  I don't believe it to be necessary even for the equivalent script in rpc-openstack.